### ISSUE TITLE
Android desugaring

### ISSUE THREAD

**Issue by robertlevonyan** [boguszpawlowski/ComposeCalendar#119]
I've updated the lib to version 1.2.0 and it started to require that the desugaring should be enabled for `java.time`. My app uses minSdk 26 so java.time should work without enabling desugaring.

here's the error
```
 Dependency 'io.github.boguszpawlowski.composecalendar:composecalendar:1.2.0' requires core library desugaring to be enabled
      for :app.
```

**Comment by voitenkodev** [comment]
the same, why is it happened? any information please

**Comment by ZhEgor** [comment]
Fixed by downgrading version from 1.2.0 to 1.1.1

**Comment by robertlevonyan** [comment]
not an option for me to downgrade

**Comment by boguszpawlowski** [comment]
Hi guys, and thank you for your patience. Will remove desugaring in 1.3.0 this week.

**Comment by boguszpawlowski** [comment]
@voitenkodev @ZhEgor @robertlevonyan  Implemented and released in 1.3.0, please let me know if everything works as expected.
